### ISSUE TITLE
does not build on windows

### ISSUE THREAD

**Issue by jstaerk** [ZUGFeRD/mustangproject#105]
...allegedly...

**Comment by swsch** [comment]
The alledged problem is probably our [old friend](https://github.com/ZUGFeRD/mustangproject/blob/24b27a9b516df3bf753119a31f0788c23b990396/src/test/java/org/mustangproject/ZUGFeRD/MustangReaderWriterCustomXMLTest.java#L67) where `new String(UTF8ByteOrderMark)` uses platform specific encodings. Either provide an explicit encoding as second argument to the string constructor or prefix the string with the character corresponding to the byte oder mark, see [this comment](https://github.com/ZUGFeRD/mustangproject/issues/99#issuecomment-511134151) and the next one.

Basically this is the same as #99

**Comment by swsch** [comment]
d47f85c should fix this issue.

Not sure why it pops up in the pull request for #23. Probably due to lack of sleep last night.

**Comment by jstaerk** [comment]
great! thanks a lot!
